xml/gl.xml

```diff
@@ -2734,9 +2734,9 @@ typedef unsigned int GLhandleARB;
         <enum value="0x84F2" name="GL_ALL_COMPLETED_NV" group="FenceConditionNV"/>
         <enum value="0x84F3" name="GL_FENCE_STATUS_NV" group="FenceParameterNameNV"/>
         <enum value="0x84F4" name="GL_FENCE_CONDITION_NV" group="FenceParameterNameNV"/>
-        <enum value="0x84F5" name="GL_TEXTURE_RECTANGLE" group="CopyImageSubDataTarget,TextureTarget"/>
-        <enum value="0x84F5" name="GL_TEXTURE_RECTANGLE_ARB"/>
-        <enum value="0x84F5" name="GL_TEXTURE_RECTANGLE_NV"/>
+        <enum value="0x84F5" name="GL_TEXTURE_RECTANGLE" group="CopyImageSubDataTarget,TextureTarget,EnableCap,GetPName"/>
```

Comment:
In what version of OpenGL is `GL_TEXTURE_RECTANGLE` a valid argument to `glEnable`? (`EnableCap`)

And why add `GetPName` to `GL_TEXTURE_RECTANGLE` when the valid enum for `glGet` is `GL_TEXTURE_BINDING_RECTANGLE`?

Reply:
I am oriented to ARB_texture_rectangle included in OpenGL 3.1
About glEnable:
```
 10) How are rectangular textures enabled?

     Rectangular textures are enabled by enabling the
     GL_TEXTURE_RECTANGLE_ARB texture target via glEnable
     (GL_TEXTURE_RECTANGLE_ARB). This enable is prioritized above
     GL_TEXTURE_2D and below GL_TEXTURE_3D.

     From lowest priority to highest priority: GL_TEXTURE_1D,
     GL_TEXTURE_2D, GL_TEXTURE_RECTANGLE_ARB, GL_TEXTURE_3D,
     GL_TEXTURE_CUBE_MAP.
```

GetPName:
```
Accepted by the <cap> parameter of Enable, Disable and IsEnabled;
    by the <pname> parameter of GetBooleanv, GetIntegerv, GetFloatv
    and GetDoublev; and by the <target> parameter of BindTexture,
    GetTexParameterfv, GetTexParameteriv, TexParameterf, TexParameteri,
    TexParameterfv and TexParameteriv:

    TEXTURE_RECTANGLE_ARB            0x84F5
```
But GetPName is strange, I can delete.